### ISSUE TITLE
Double folders/ path

### ISSUE THREAD

**Issue by gregseed** [terraform-google-modules/terraform-google-iam#65]
I just tried to run folder permissions and got the following:

```
Error retrieving IAM policy for folder "folders/folders/XXXXXXXXXX": googleapi: got HTTP response code 404 with body: <!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Not Found)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>404.</b> <ins>That’s an error.</ins>
  <p>The requested URL <code>/v2/folders/folders/XXXXXXXXXX:getIamPolicy?alt=json&amp;prettyPrint=false</code> was not found on this server.  <ins>That’s all we know.</ins>


  on .terraform\modules\iam_folder.iam\terraform-google-modules-terraform-google-iam-ace2efe\modules\folders_iam\main.tf line 44, in resource "google_folder_iam_member" "folder_iam_additive":
  44: resource "google_folder_iam_member" "folder_iam_additive" {
```

Scripts

**`./main.tf`**

```hcl
provider "google" {
  version = "~> 2.17"
  region      = "${var.gcp_region}"
  credentials = "${file("${var.gcp_credentials}")}"
}

resource "google_folder" "the_folder" {
  display_name = "the-folder"
  parent       = "XXXXXX"
}

module "iam_folder" {
  source         = "./modules/folder-iam-policy"
  folder_ids     = [google_folder.the_folder.id]
  folders_num    = 1
  editor_members = ["xxxx@xxxx.xx"]
  admin_members  = ["xxxx@xxxx.xx"]
  viewer_members = ["xxxx@xxxx.xx"]
}
```

**`./modules/folder-iam-policy/main.tf`**
```hcl
module "iam" {
  source = "terraform-google-modules/iam/google//modules/folders_iam"
  version = "~> 3.0"
  folders = var.folder_ids
  folders_num = var.folders_num
  bindings = {
    # explicit viewer_members
    "roles/compute.instanceAdmin"          = var.admin_members
    "roles/bigquery.admin"                 = var.admin_members
    "roles/resourcemanager.folderAdmin"    = var.admin_members
    "roles/resourcemanager.folderIamAdmin" = var.admin_members

    
    "roles/resourcemanager.folderViewer"       = flatten([var.editor_members, var.viewer_members])
  }
}
```

**`./modules/folder-iam-policy/variables.tf`**
```hcl
variable admin_members {
  default = []
  type    = list(string)
}
variable viewer_members {
  default = []
  type    = list(string)
}
variable editor_members {
  default = []
  type    = list(string)
}

variable folder_ids {
  default = []
  type    = list(string)
} 

variable folders_num {
  default = 0
}
```

**Comment by aaron-lane** [comment]
Hi @gregseed. Thank you for the bug report.

As a work around, can you try `folder_ids = [replace(google_folder.the_folder.id, "folders/", "")]`?

**Comment by jason-huling** [comment]
The above work around worked for me when I ran into this. 

@aaron-lane are there any plans to address this without the workaround, or could you do this internally in the module? Especially considering that the `google_folder` resource doesn't have any output attribute without the `folders/`prefix.

**Comment by morgante** [comment]
Agreed, we should fix this in the module by automatically stripping the `folders/` prefix if it was supplied. Should be a simple fix.

**Comment by aaron-lane** [comment]
Fixed by #77.
